### ISSUE TITLE
[BUG] Latest SteamVR beta won't launch (missing file?)

### ISSUE THREAD

**Issue by Danielduel** [ValveSoftware/SteamVR-for-Linux#599]
Hi, the latest SteamVR Beta tries to launch a file that doesn't exist
```
/home/daniel/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/SteamVR/bin/vrstartup.sh: line 3: ./vrsetup.sh: No such file or directory
+ exec ./vrstartup-internal.sh
/home/daniel/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/SteamVR/bin/vrstartup.sh: line 5: /home/daniel/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/SteamVR/vrstartup-internal.sh: No such file or directory
```
SteamVR in both - flatpak and standard version of steam client has this problem.

The "standard" version can't see 1.27.1 in patch notes, but selecting beta still updates the steamvr to beta depot (steam api bug?)

I've tried reinstalling SteamVR inside both steam clients - beta always fails to start.

---
I think in this case the usual form is not needed, but for completeness sake:

**Describe the bug**
Latest SteamVR (1.27.1) won't launch. 

**To Reproduce**
Try launching steamvr by "Launch" in steam client.

**Expected behavior**
SteamVR launching.

**System Information (please complete the following information):**
- Distribution: Arch
- SteamVR version: 1.27.1
- Steam client version: 1690583737
- Opted into Steam client beta?: Yes
- Graphics driver version:  latest mesa
- Gist for SteamVR System Information: -

**Comment by apcolvin** [comment]
I have the exact same issue

**Comment by VRcontrib** [comment]
Same issue here!

**Comment by wallcarpet40** [comment]
Yup, 1.27.1 broke the whole SteamVR beta.

**Comment by Xanadu-Zen** [comment]
Just copy vrstartup.sh from old version of SteamVR and overwrite it can temporary fix this issue

**Comment by lightstream** [comment]
> Just copy vrstartup.sh from old version of SteamVR and overwrite it can temporary fix this issue

This does seem to allow the beta to run, but then the System buttons on my Vive controllers don't work and I can only quit the game via the desktop. Also Halflife Alyx won't start, complaining about Mongoose timing out, whatever that is. I guess I'll stick with the stable version for now.

**Comment by Danielduel** [comment]
@Xanadu-Zen thank you for this workaround, but this issue is less of "I want to run beta" and more like a red alarm that they forgot to package a file ^^

**Comment by WaviestBalloon** [comment]
Thought I'd drop in to add this timestamp: https://youtu.be/YVkbcZ8JH2E?t=73

**Comment by MadByteDE** [comment]
Same here. :)

**Comment by kisak-valve** [comment]
Hello, per "Fixed missing startup script on Linux." in the [SteamVR beta 1.27.2](https://store.steampowered.com/news/app/250820/view/6575871026909712810) update, please retest this issue.

**Comment by wallcarpet40** [comment]
SteamVR 1.27.2 beta fixes this issue for me. Fedora 38.

**Comment by Danielduel** [comment]
Fixed in 1.27.2
